source/fundamentals/run-command.txt

```diff
@@ -55,17 +55,55 @@ the current member's role in the replica set, on a database:
 For a full list of database commands and corresponding parameters, see
 the :ref:`Additional Information section <addl-info-runcommand>`.
 
+.. _node-command-options:
+
+Command Options
+---------------
+
+You can specify optional command behavior for the ``command()``
+and ``runCursorCommand()`` methods.
+
+The ``command()`` method accepts a ``RunCommandOptions`` object. To learn
+more about the ``RunCommandOptions`` type, see the `API documentation <{+api+}/types/RunCommandOptions.html>`__.
+
+The ``runCursorCommand() method`` accepts a ``RunCursorCommandOptions``
+object. To learn more about the ``RunCursorCommandOptions`` type, see
+the `API documentation <{+api+}/types/RunCursorCommandOptions.html>`__.
+
+Starting in version 6.0 of the {+driver-short+}, you can pass only the
+following options directly to the ``command()`` method:
+
+- ``comment``
+- ``enableUtf8Validation``
+- ``raw``
+- ``readPreference``
+- ``session``
+
+You can set additional options, including a write or read
+concern, in the command document that you pass to
+the ``command()`` method. See the Server manual :manual:`entry
+</reference/command/>` for a command to learn about its associated
```

Comment:
Issues:

1) I think this could be confusing to a user since the page this links to shows the overview of runCommand() and adminCommand() rather than command() and "its associated options". I also wasn't able to locate the aforementioned write or read concern options.

2) The text of the link should be descriptive. See [Link and Cross-Reference Examples](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-examples).

Suggestion:

From looking at docs/manual/reference/command, it looks like the options are listed only on the target pages of the links of each command. It could be helpful to inform the reader that is how it is structured to reduce the chance of confusion. Also, I think it would be good to continue using the format "\<Page title\>" in the Server Manual.

E.g.
"Learn more about the available options for each command by following the link associated with the command in the :manual:\`User Commands </reference/command/#user-commands>\` section of the Server Manual"

Reply:
since the page describes commands outside of the User Commands category (such as `hello`), I will change the link and link text slightly

Reply:
Resolving and adding a comment to the revised version.